components/bms_worker/src/BatteryMonitoring.c

```diff
@@ -33,11 +33,11 @@
 #endif
 
 #ifndef STANDARD_CHARGE_CURRENT
-# define STANDARD_CHARGE_CURRENT 4.2    // in Amps
+# define STANDARD_CHARGE_CURRENT 4.2f
```

Comment:
Why remove the comment with the unit? These are typically good to have, though I prefer this format:
```suggestion
# define STANDARD_CHARGE_CURRENT 4.2f // [A] Standard charge current
```
Might be better to use the formatting that Doxygen wants